### PR TITLE
minor changes and typo correction

### PR DESCRIPTION
**Pull request by MOON-CLJ**
cc @everpcpc

### CODE REVIEW

File: go/logic/applier.go [hunk 1]
```diff
@@ -1001,15 +1002,19 @@ func (this *Applier) ApplyDMLEventQuery(dmlEvent *binlog.BinlogDMLEvent) error {
 			if err != nil {
 				return err
 			}
+			rollback := func(err error) error {
+				tx.Rollback()
+				return err
+			}
```
Comment (MOON-CLJ) [inline go/logic/applier.go]: @shlomi-noach just make sure "A transaction must end with a call to Commit or Rollback." and consistent with ApplyDMLEventQueries

File: go/logic/applier.go [hunk 2]
```diff
@@ -482,6 +482,7 @@ func (this *Applier) ApplyIterationInsertQuery() (chunkSize int64, rowsAffected
 		if err != nil {
 			return nil, err
 		}
+		defer tx.Rollback()
```
Comment (MOON-CLJ) [inline go/logic/applier.go]: @shlomi-noach just make sure "A transaction must end with a call to Commit or Rollback."

File: go/logic/inspect.go [hunk 3]
```diff
@@ -701,14 +701,17 @@ func (this *Inspector) getSharedColumns(originalColumns, ghostColumns *sql.Colum
 		for _, ghostColumn := range ghostColumns.Names() {
 			if strings.EqualFold(originalColumn, ghostColumn) {
 				isSharedColumn = true
+				break
```
Comment (MOON-CLJ) [inline go/logic/inspect.go]: add these breaks make no changes。 no need to continue the whole for loop?

File: go/sql/builder.go [hunk 4]
```diff
@@ -140,13 +140,12 @@ func BuildRangeComparison(columns []string, values []string, args []interface{},
...
 
 	for i, column := range columns {
-		//
 		value := values[i]
 		rangeComparison, err := BuildValueComparison(column, value, comparisonSign)
 		if err != nil {
 			return "", explodedArgs, err
 		}
-		if len(columns[0:i]) > 0 {
+		if i > 0 {
```
Comment (MOON-CLJ) [inline go/sql/builder.go]: equivalent and clear?

**Comment by shlomi-noach** [comment]
Thank you again, looking good. It will take me some time to merge this.

**Comment by MOON-CLJ** [comment]
@shlomi-noach thx，i just done "rebase -i master"

**Pull request merged**